src/index.js

```diff
@@ -0,0 +1,59 @@
+const isTest = process.env.NODE_ENV === 'test';
+
```

Comment:
long-term -- eventually use the same `NODE_ENV` check that's inside `cnn-server` -- we'll need to break that logic out into a separate package